### ISSUE TITLE
Poor update check UI

### ISSUE THREAD

**Issue by AGSPhoenix** [duplicati/duplicati#3188]
- [x] I have searched open and closed issues for duplicates.
----------------------------------------
## Environment info

- **Duplicati version**: 2.0.3.3_beta_2018-04-02

## Description

tl;dr at the end
The UI for checking for updates is poor. This picture shows the part of the UI you click to check for updates:
![image](https://user-images.githubusercontent.com/450785/39095724-1257d3de-4613-11e8-8904-98aa4063b0b4.png)
Where is it? The buttons at the top are tabs, and the first few links are boring stuff about the people who made it (sorry) and legal stuff, and the rest of the text is not interactive and therefore not an update button. Clicking through the tabs, none of them have an update button in them either (and the last one is not a tab but a link to another part of the UI despite being visually identical, but that's another issue).

The answer: it's under the General tab where we started, as a link. A link that is right under a bunch of other links that immediately get classified as useless and uninteresting (again, sorry) when skimming.

After clicking the link, it gets replaced with the text `Checking for updates ...`, which promptly gets replaced with the update link again, with no indication of whether or not there are updates, or if there was a problem checking for updates.

tl;dr: The check for updates link should be a button, and should give feedback about what it finds.

## Screenshots
I made a mockup of a somewhat better interface (though it doesn't match the styling of the rest of the UI; my CSS is not great):
![image](https://user-images.githubusercontent.com/450785/39095998-83203f44-4617-11e8-8d1b-b5ff84b31f87.png)
Clicking it shows the status next to the button:
![image](https://user-images.githubusercontent.com/450785/39096012-b7427922-4617-11e8-8702-b7594837d1aa.png)
No updates shows something like this (green checkmark looks kinda over the top, but more UI-appropriate color coding is probably a good idea):
![image](https://user-images.githubusercontent.com/450785/39096055-72199866-4618-11e8-85c0-1167ba9803a3.png)
And errors (again, some color coding here might be a good idea):
![image](https://user-images.githubusercontent.com/450785/39096073-b0778460-4618-11e8-9ad4-12805f5ab20b.png)

**Comment by piegamesde** [comment]
Before I used Duplicati from the AUR I had to search for the update button every single time.

**Comment by SanduRajapakse** [comment]
Good Suggestion... I will see if the backend can support this.
